local-requirements.txt

```diff
@@ -20,3 +20,4 @@ service_identity==24.2.0
 twisted==24.11.0
 types-pyOpenSSL==24.1.0.20240722
 types-requests==2.32.0.20241016
+uv==0.5.4
```

Comment:
I'd prefer to not introduce a new package for this. What do we loose if we don't use it? afaik its only used to update the command in the `requirements.txt`?

Reply:
I needed something to compile the requirements, it's either this `uv` or `pip-tools` or maybe `poetry` (?). Unless `conda` has options to do this but I think `conda` is slowing down, and `uv` is the future of package management in python, replacing/improving-on/consolidating `pip`, `virtualenv`, `pyenv`, `conda`, ...

Reply:
Here https://chatgpt.com/share/6764e629-4344-8010-8465-c1efe5e1813e

Reply:
I see - so both dependencies: `pyee` and `greenlet` have no dependencies on itself - this is where I got confused. uv helps us to pin the transitive dependencies (where there aren't any yet). In the future there might be some. Is this correct?

Reply:
that's right, generally good to have pin deps for CI otherwise a new lib comes out some day and your CI starts failing.